src/components/earlyadopter/ConnectAccount.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="pt-20 pb-20 status-background">
+    <div class="max-w-6xl mx-auto">
+      <div class="justify-center">
+        <h1 class="font-bold text-xl">
+          Early adopter participation confirmation
+        </h1>
+        <div>
+          <div>
+            <div
+              class="
+                mt-1
+                w-full
+                rounded-md
+                bg-white
+                dark:bg-darkGray-900
+                border border-gray-300
+                dark:border-darkGray-500
+              "
+            >
+              <div
+                v-if="!showConnectMetamsk"
+                class="transform text-gray-400 dark:text-darkGray-500"
+              >
+                Loading....
+              </div>
+
+              <h3
+                v-if="showConnectMetamsk"
+                class="
+                  text-lg
+                  font-extrabold
+                  text-blue-900
+                  dark:text-white
+                  mb-6
+                  text-center
+                "
+              >
+                Choose Account that you want to apply the lockdrop bonus.
+              </h3>
+              <ul
+                class="
+                  max-h-56
+                  rounded-md
+                  py-1
+                  text-base
+                  overflow-auto
+                  focus:outline-none
+                "
+              >
+                <AccountOption
```

Comment:
![Screen Shot 2021-12-20 at 12 10 41 PM](https://user-images.githubusercontent.com/40356749/146758435-57474e4a-8585-43a4-9b31-6b1adfac3852.png)
The properties should be in a straight line (the position of the 'DOT' text)